### PR TITLE
Altering /clients/connections with a temporary multitenancy scenario

### PR DESCRIPTION
**Pull request by jeffreylees**
https://trello.com/c/UYWX9NJf

Replacing recommendation to create separate connections for separate clients in the same tenant with a very basic multi-tenancy scenario. This section will eventually be replaced with a link to a more in-depth multi-tenancy document with an example and diagram.

https://auth0-docs-content-pr-5792.herokuapp.com/docs/clients/connections

### CODE REVIEW

File: articles/clients/connections.md [hunk 1]
```diff
@@ -6,37 +6,24 @@ crews: crew-2
...
-- For the `Fabrikam Employee Timesheets` client enable the Active Directory / LDAP and Google Apps connections.
-- For the `Fabrikam Customer Portal` client enable the ADFS, Microsoft Azure AD, Google Apps and LinkedIn connections.
-- That's it!
+
+- Create a tenant `Fabrikam-Internal` for your internal domain, and a client within it `Fabrikam Employee Timesheets` for timesheets.
+- Create a second tenant `Fabrikam-Public` for your public-facing domain, and a client within it `Fabrikam Customer Portal` for the customer portal.
+- Configure the following [Enterprise connections](${manage_url}/#/connections/enterprise) for the `Fabrikam-Internal` tenant: Active Directory / LDAP, and Google Apps; once each is set up, check the **Clients** tab to enable them.
+- Configure the following [Enterprise connection](${manage_url}/#/connections/enterprise) for the `Fabrikam-Public` tenant: Google Apps; once it is set up, check the **Clients** tab to enable it.
+- Configure the following [Social connections](${manage_url}/#/connections/social) for the `Fabrikam-Public` tenant: Facebook, LinkedIn; once each is set up, check the **Clients** tab to enable it.
```
Comment (mpaktiti) [inline articles/clients/connections.md:29]: Here we say to use the Clients tab to enable/disable a connection but we removed the guidance that says how to do this earlier in the doc.

**Review by mpaktiti — CHANGES_REQUESTED** [review]

File: articles/clients/connections.md (hunk 1 above)
Comment (jeffreylees) [reply articles/clients/connections.md:29]: Responded on Slack... TL;DR is that those were are separate places, and unfortunately, with the dashboard UI as it is currently, you have to toggle the clients on. The idea is that you should toggle them all on, I guess.

**Comment by jeffreylees** [comment]
@mpaktiti let's go ahead and merge this unless you have anything else for me to change?

**Comment by jeffreylees** [comment]
@mpaktiti ping

**Pull request merged**